configs/partial_conv/pconv_stage1_8xb1_places-256x256.py

```diff
@@ -54,7 +54,7 @@
 test_pipeline = train_pipeline
 
 train_dataloader = dict(
-    batch_size=1,
+    batch_size=12,
```

Comment:
You need to rename this file from `8xb1` to `8xb12`. 
Please also check previous configs to align the setting and the filename.